### PR TITLE
feat(jest): 17% coverage for PhantomManager

### PR DESCRIPTION
**Pull request by drepram**
**What this PR does** 📖

Self-explanatory. Improvements to come as Drew will change the wallet flow.

**Which issue(s) this PR fixes** 🔨

AP-2142

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

File: libraries/Phantom/PhantonManager.test.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    } catch (error) {
+      expect(error).toBeInstanceOf(Error)
+      expect(error).toHaveProperty(
+        'message',
+        AccountsError.MNEMONIC_NOT_PRESENT,
+      )
+    }
+  })
+})
+describe('Test PhantomManager after Solana mock', () => {
```
Comment (stavares843) [inline libraries/Phantom/PhantonManager.test.ts:18]: can you add a empty line before this describe for better read?

File: libraries/Phantom/PhantonManager.test.ts [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+  const original = window.solana
+
+  beforeAll(() => {
+    Object.defineProperty(window, 'solana', {
```
Comment (stavares843) [inline libraries/Phantom/PhantonManager.test.ts:22]: Captura de ecrã 2022-08-24, às 17 52 26

can we get rid of this warning?

File: libraries/Phantom/PhantonManager.test.ts [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+        AccountsError.MNEMONIC_NOT_PRESENT,
+      )
+    }
+  })
+
+  it('should fail to initialize wallet', async () => {
+    // Reason for failing: related to non-existence of isPhantom on window.solana
+    try {
+      const result = new PhantomManager()
+      result.getAccountBalance(web3.Keypair.generate())
```
Comment (stavares843) [inline libraries/Phantom/PhantonManager.test.ts:54]: Captura de ecrã 2022-08-24, às 17 52 50

can we fix this warning?

File: libraries/Phantom/PhantonManager.test.ts (hunk 1 above)
Comment (drepram) [reply libraries/Phantom/PhantonManager.test.ts]: Yes, please check :D

File: libraries/Phantom/PhantonManager.test.ts (hunk 2 above)
Comment (drepram) [reply libraries/Phantom/PhantonManager.test.ts:21]: Yes, please check :D

File: libraries/Phantom/PhantonManager.test.ts (hunk 3 above)
Comment (drepram) [reply libraries/Phantom/PhantonManager.test.ts]: Yes, please check :D

File: libraries/Phantom/PhantonManager.test.ts (hunk 2 above)
Comment (drepram) [reply libraries/Phantom/PhantonManager.test.ts:21]: (See my Discord DM on what Zacca's opinion on it)

Comment (stavares843) [reply libraries/Phantom/PhantonManager.test.ts:21]: thanks 🔨

**Review by stavares843 — APPROVED** [review]

**Review by JustZacca — APPROVED** [review]

**Pull request merged**